include/vcpkg/paragraphparser.h

```diff
@@ -40,10 +40,19 @@ namespace vcpkg
         std::vector<LocalizedString> errors;
     };
 
+    Optional<std::vector<std::string>> parse_default_features_list_context(DiagnosticContext& context,
+                                                                           const std::string& str,
+                                                                           StringView origin = "<unknown>",
```

Comment:
I have a different take: `<unknown>` must not exist in the first place, and anywhere it can happen is a bug. But this PR is big enough without trying to solve for that problem...

Reply:
> `<unknown>` must not exist in the first place

And now it doesn't https://github.com/microsoft/vcpkg-tool/pull/1346